src/benchmark/join_order_benchmark.cpp

```diff
@@ -113,5 +115,48 @@ int main(int argc, char* argv[]) {
   auto query_generator =
       std::make_unique<FileBasedQueryGenerator>(*benchmark_config, query_path, non_query_file_names, query_subset);
 
-  BenchmarkRunner{*benchmark_config, std::move(query_generator), std::move(table_generator), context}.run();
+  auto benchmark_runner =
+      BenchmarkRunner{*benchmark_config, std::move(query_generator), std::move(table_generator), context};
+
+  if (benchmark_config->verify) {
+    // Add indexes to SQLite. This is a hack until we support CREATE INDEX ourselves and pass that on to SQLite.
+    // Without this, SQLite would never finish.
+
+    std::cout << "- Adding indexes to SQLite" << std::endl;
+    Timer timer;
+
+    // SQLite does not support adding primary keys, so we rename the table, create an empty one from the provided
+    // schema and copy the data.
+    for (const auto& table_name : StorageManager::get().table_names()) {
+      benchmark_runner.sqlite_wrapper->raw_execute_query(std::string{"ALTER TABLE "} + table_name +  // NOLINT
+                                                         " RENAME TO " + table_name + "_unindexed");
+    }
+
+    // Recreate tables from schema.sql
+    std::ifstream schema_file(std::string{DEFAULT_QUERY_PATH} + "/schema.sql");  // NOLINT
```

Comment:
why not use `query_path` path here - `--query_path` is an option and should probably be respected. (useful, e.g., for running the JOB from the build dir)